README.md

```diff
@@ -1 +1,33 @@
-# CSV-Users
\ No newline at end of file
+# CSV-Users
+
+This package is responsible for exporting and importing users in `.csv` format
+
+[![swagger](https://img.shields.io/badge/documentation-swagger-green)](https://escolalms.github.io/CSV-Users/)
+[![codecov](https://codecov.io/gh/EscolaLMS/CSV-Users/branch/main/graph/badge.svg?token=NRAN4R8AGZ)](https://codecov.io/gh/EscolaLMS/CSV-Users)
+[![phpunit](https://github.com/EscolaLMS/CSV-Users/actions/workflows/test.yml/badge.svg)](https://github.com/EscolaLMS/CSV-Users/actions/workflows/test.yml)
+[![downloads](https://img.shields.io/packagist/dt/escolalms/CSV-Users)](https://packagist.org/packages/escolalms/CSV-Users)
+[![downloads](https://img.shields.io/packagist/v/escolalms/CSV-Users)](https://packagist.org/packages/escolalms/CSV-Users)
+[![downloads](https://img.shields.io/packagist/l/escolalms/CSV-Users)](https://packagist.org/packages/escolalms/CSV-Users)
+[![Maintainability](https://api.codeclimate.com/v1/badges/04a88ff03ede597fd18b/maintainability)](https://codeclimate.com/github/EscolaLMS/CSV-Users/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/04a88ff03ede597fd18b/test_coverage)](https://codeclimate.com/github/EscolaLMS/CSV-Users/test_coverage)
+
+
+## Installation
+
+```
+composer require escolalms/csv-users
+```
+
+## API
+
+There are two endpoints defined in this package.
```

Comment:
it's already described here https://escolalms.github.io/CSV-Users/#/Admin%20CSV/5e29d0712a6da223f3d0e92738ef3ccd :D